### PR TITLE
Updated README with binary directory structure

### PR DESCRIPTION
**Pull request by bobhancock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,72 @@
...
-To run the program, you first need to install the Go programming language
-runtime. Download the [latest version](https://golang.org/dl/) and follow
-the installation instructions.
+# Downloads
+If you are building from source, follow the
+[source installation instructions](#source).
 
-Clone the repository outside of your GOPATH. If you must clone within your GOPATH
-then set the environment variable `GO111MODULE=off`
+If you want to use a binary, fetch the repository and
```
Comment (bonniecpk) [inline README.md]: I suggest renaming this sentence to

"If you want to execute the program directly, clone the repository..."

File: README.md [hunk 2]
```diff
@@ -1,68 +1,72 @@
...
-Once you have verified your Go installation, in a terminal, change to the
-directory where you cloned the repository. Change to the directory
-oauthdoctor.
+```
+cd <path>/google-ads-doctor/oauthdoctor/bin
+```
 
-At the command line, type:
+There are three directories, one each for Darwin (OSX), Linux, and Windows.
+You can run the binary in place or copy the file to any location on your system.
```
Comment (bonniecpk) [inline README.md]: Suggest rewriting the sentence to:

"You can run the executable file in place or..."

File: README.md (hunk 1 above)
Comment (bobhancock) [reply README.md]: I know what you are getting at, I think:

I you want use a pre-complied executable file.

The second phrase is unclear, we should just point them to the bin section.

File: README.md (hunk 2 above)
Comment (bobhancock) [reply README.md]: I searched for other project and they refer to these files as binaries and not executables.

https://golang.org/dl/ refers to the pre-build releases as binaries.

File: README.md (hunk 1 above)
Comment (bobhancock) [reply README.md]: f you want to use a pre-compiled binary, fetch the repository and

**Pull request merged**